challenge-03/qosf-monthly-challenge-03.ipynb

```diff
@@ -0,0 +1,56 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Challenge 03: Minimum Makespan with Quantum Annealing\n",
+    "\n",
+    "Niranjan owns a catering service and receives orders one day in advance. Each of the chefs he employs is payed by the hour. In order to optimize the cost of running his business, every night, he looks at all of the dishes he needs to deliver for the next day and asks his employees to work the specific hours he requires them for.\n",
+    "\n",
+    "Given an order queue of length $N$ with each order $i$ taking $L_i$ time to prepare, if there are $m$ chefs, design a QUBO/Ising Hamiltonian to help Niranjan find the lowest time it would take for the $m$ chefs to prepare all $N$ dishes. \n",
```

Comment:
```suggestion
    "Given an order queue of length $N$ with each order $i$ taking $L_i$ time to prepare, if there are $m$ chefs, design a QUBO/Ising Hamiltonian to help Niranjan find the shortest time it would take for the $m$ chefs to prepare all $N$ dishes. \n",
```

Reply:
I've never seen this type of problem before, so bear with me. A few questions:
1. Are all orders the same type?
2. Does each chef have equal capabilities in cooking each meal?
3. Are there any limits on how many chefs can cook at the same time?

Can you provide sample input data and output data to aid the participant in knowing correct output?

Reply:
Hi,

This is also my first time seeing this type of problem, so I also have some questions:
1. I am not familiar with QUBO at all. Is there a gentle introduction to the algorithm? I see you provided some references, but they are very lengthy and use very advanced vocabulary. Is there something we can add that guides the reader from basic understanding of quantum algorithms to a full implementation of QUBO for a specific problem?

2. Is the idea to have participants solve problem on quantum annealer? Or can a universal QC be used too? We might need to clarify this, and also provide references on how this can be done using the latter.

Reply:
These are all important points – I think the answer for these answers to all of them should be "yes", as otherwise you end up with a much more complicated version of this problem.

Reply:
@amirebrahimi Yes, I've added these clarifications to the questions. Thanks. 

@diemilio AFAIK, we would have to use a quantum annealer to solve these problems. Does the intro that @mstechly pointed below work?